image.yaml

```diff
@@ -68,14 +72,14 @@ modules:
           - name: os-eap7-openshift
           - name: os-eap-s2i
           - name: os-java-jolokia
-          - name: jolokia
+          - name: jboss.eap.cd.jolokia
           - name: os-eap7-openshift
-          - name: os-eap7-modules
+          - name: jboss.eap.cd.openshift.modules
```

Comment:
You may want to include the jboss-eap-config-jgroups and elytron modules as well, like here:

https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap-cd/image.yaml#L56

These update the elytron and jgroups configs to use a non-deprecated configuration.

Reply:
@luck3y Thanks, Ken! Included those (together with the metrics & tracing ones), see last commit.